explorer/models/rootNode.ts

```diff
@@ -108,16 +105,15 @@ export class RootNode extends NodeBase {
 
     }
 
-    public async getChildren(element: NodeBase): Promise<NodeBase[]> {
-
+    public async getChildren(element: RootNode): Promise<NodeBase[]> {
         if (element.contextValue === 'imagesRootNode') {
```

Comment:
As a note for equality cases we usually use switch statements, alternatively use elseif to avoid reevaluation.